app.js

```diff
@@ -28,6 +28,12 @@ app.locals.title = `${capitalized(projectName)}- Generated with IronGenerator`;
 const index = require("./routes/index");
 app.use("/", index);
 
+const celebrities = require('./routes/celebrities');
+app.use('/', celebrities);
```

Comment:
I would call the base route "/celebrities" here, and code the continuation of that base path in the "celebrities.js" file. Same goes for the movies route.